gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -137,11 +137,22 @@
               </div>
             </div>
           </div>
-          <div class="row form-group">
-            <div class="col-md-12">
+          <div class="row">
+            <div class="col-md-6" ng-class="{'has-error': deviceEditor.numConnections.$invalid && deviceEditor.numConnections.$dirty}">
+              <label translate>Connection Management</label>
+              <div class="row">
+                <span class="col-md-8" translate>Number of Connections</span>
+                <div class="col-md-4">
+                  <input name="numConnections" id="numConnections" class="form-control" type="number" pattern="\d+" ng-model="currentDevice.numConnections" min="0" />
+                </div>
+              </div>
+              <p class="help-block" ng-if="!deviceEditor.numConnections.$valid && deviceEditor.numConnections.$dirty" translate>The number of connections must be a non-negative number.</p>
+              <p class="help-block" ng-if="deviceEditor.numConnections.$valid || deviceEditor.numConnections.$pristine" translate>When set to more than one, Syncthing will attempt to establish multiple concurrent connections. One connection is reserved for control messages, the rest are used for data transfer. When set to one, the single connection is used for both. Set to zero to let Syncthing decide.</p>
```

Comment:
```suggestion
              <div class="help-block" ng-if="deviceEditor.numConnections.$valid || deviceEditor.numConnections.$pristine">
                <p translate>When set to more than one, Syncthing will attempt to establish multiple concurrent connections. For this to succeed, the number must be set to the same value on both devices.</p>
                <p translate>One connection is reserved for control messages, the rest are used for data transfer. When set to one, the single connection is used for both. Set to zero to let Syncthing decide.</p>
              </div>
```

I've added "For this to succeed, the number must be set to the same value on both devices." as I think this is crucial information here, and I've also split the text into two paragraphs, as it seemed a bit too long (and problematic when translating when changing a single word would force to re-translate the whole thing).

Reply:
It doesn't have to be the same value, though. I think this is already too long to be the description text here. How about walking back a lot of this to just something like `How many parallel connections to use with this device, or zero to let Syncthing decide.` and the usual help icon that points to the documentation article?

Reply:
Honestly, I'm not a fan of moving everything to the Docs, because a) the Docs tend to be much more technical, and b) the Docs are English-only.

How about a shorter version, e.g. like this:

```suggestion
              <p class="help-block" ng-if="deviceEditor.numConnections.$valid || deviceEditor.numConnections.$pristine" translate>When set to more than one, Syncthing will attempt to establish multiple concurrent connections. For this to work, the number must be at least equal on both devices. Set to zero to let Syncthing decide.</p>
```

Reply:
It doesn't *have to* be at least equal either, it'll work anyway just with fewer connections and now it sounds threatening :D

Sorry, I realise it's really tricky to be concise and yet convey the desirable meaning. Im not sure what to say here.

Reply:
Understood 😉. Yet another try:

```suggestion
              <p class="help-block" ng-if="deviceEditor.numConnections.$valid || deviceEditor.numConnections.$pristine" translate>When set to more than one, Syncthing will attempt to establish multiple concurrent connections. If not equal on both devices, the lower value will be used. Set to zero to let Syncthing decide.</p>
```

Reply:
It's actually the higher value, if both sides are set to a value higher than one, otherwise one... 🥲 But... I wouldn't be surprised if this is one of those things we change when this gets deployed...

Reply:
Is it like that? I mean, if one device is set to `2` and the other to `3`, what happens then?

Reply:
Three

Reply:
I see… Is it safe like this? For example, if I'm connected to someone else's device (e.g. a friend), and they set the number of connections to a crazy value like `1000000` (let's say just by mistake), will this essentially break Syncthing on my side?

Reply:
Probably! Maybe we should have an upper limit.

Reply:
```suggestion
              <p class="help-block" ng-if="deviceEditor.numConnections.$valid || deviceEditor.numConnections.$pristine" translate>When set to more than one on both devices, Syncthing will attempt to establish multiple concurrent connections. If the values differ, the highest will be used. Set to zero to let Syncthing decide.</p>
```

Reply:
I added a limit below.